Models/ConfigurationModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace SmartStore.SwissBitcoinPay.Models
+{
+    [LocalizedDisplay("Plugins.Smartstore.SwissBitcoinPay.")]
+    public class ConfigurationModel : ModelBase
+    {
+
+        [LocalizedDisplay("*ApiUrl")]
+        [Url]
+        //[Required]
+        public string ApiUrl { get; set; }
+
+        [LocalizedDisplay("*ApiKey")]
+        //[Required]
+        public string ApiKey { get; set; }
+
+        [LocalizedDisplay("*ApiSecret")]
+        //[Required]

```

Comment:
ApiSecret should not be required.
If not defined, the way to secure the webhook is as follow:
1. In the webhook handler, make a GET request to `/checkout/${invoiceId}`
2. Check if the `isPaid` value is effectively `true`
3. Set the order as Paid

Reply:
Can we discuss about this ?